python/lsst/ts/salobj/sal_info.py

```diff
@@ -236,6 +236,7 @@ def __init__(self, domain, name, index=0):
             raise TypeError(f"domain {domain!r} must be an lsst.ts.salobj.Domain")
         self.isopen = True
         self.domain = domain
+        self.debug = False
```

Comment:
Consider not using this attribute and determining if running in debug mode from the log level. I think it will make the code cleaner.

Reply:
That was debugging code accidentally left in. I will remove it.